### PR TITLE
feat(SGC-143): display delivered status for Govsg campaigns

### PR DESCRIPTION
**Pull request by tan-zx**
Took the liberty to:
- Clean up some parts of the code
- Reorder the status so that it follows a more logical order 

Deployment notes:
- [x] Tested on staging
- [ ] Run db migration
- [ ] Deploy backend

BEFORE:

image

AFTER:

image

### CODE REVIEW

File: frontend/src/components/common/progress-details/ProgressDetails.tsx [hunk 1]
```diff
@@ -192,10 +192,22 @@ const ProgressDetails = ({
             <td className={'md'}>Sent to recipient</td>
             <td className={'sm'}>{sent}</td>
           </tr>
+          {type === ChannelType.Govsg && (
+            <tr>
+              <td className={cx(styles.status, 'md')}>
+                <i
+                  className={cx(styles.icon, styles.grey, 'bx bx-check-double')}
```
Comment (stanleynguyen) [inline frontend/src/components/common/progress-details/ProgressDetails.tsx:199]: Can we use `bx-mail-send` for delivered instead?

**Review by stanleynguyen — APPROVED** [review]

File: frontend/src/components/common/progress-details/ProgressDetails.tsx (hunk 1 above)
Comment (tan-zx) [reply frontend/src/components/common/progress-details/ProgressDetails.tsx:199]: I was thinking it looks like the grey double ticks on WhatsApp lol. but OK either way.

Comment (tan-zx) [reply frontend/src/components/common/progress-details/ProgressDetails.tsx:199]: Kendra has added to the slides alr — let's just merge as-is, if you don't mind terribly?

Comment (stanleynguyen) [reply frontend/src/components/common/progress-details/ProgressDetails.tsx:199]: Yea ok sorry let's go

**Pull request merged**